docs/function-controller-configuration.md

```diff
@@ -227,14 +227,16 @@ The same process should be followed for any trigger controller installed (Kafka,
 
 ## Using custom images
 
-It is possible to configure the different images that Kubeless uses for deploy and execute functions. In this ConfigMap you can configure:
+It is possible to configure the different images that Kubeless uses to deploy and execute functions. In this ConfigMap you can configure:
 
- - Different or additional runtimes. For doing so it is possible to modify/add a runtime in the field `runtimeImages`. Runtimes are categorized by major version. See the guide for [implementing a new runtime](/docs/implementing-new-runtime) for more information. Each major version has:
+ - Different or additional runtimes. For doing so it is possible to modify/add a runtime in the field `runtime-images`. Runtimes are categorized by major version. See the guide for [implementing a new runtime](/docs/implementing-new-runtime) for more information. Each major version has:
   - Name: Unique ID of the runtime. It should contain the runtime name and version.
   - Version: Major and minor version of the runtime.
   - Runtime Image: Image used to execute the function.
   - Init Image: Image used for installing the function and/or dependencies.
   - (Optional) Image Pull Secrets: Secret required to pull the image in case the repository is private.
+  - (Optional) Environment variables.
+  - (Optional) Secrets: Shared with the container as mounted volumes.
```

Comment:
I would add here the path in which the secrets are mounted.

Reply:
Added